src/redux/store.jsx

```diff
@@ -0,0 +1,13 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] @Team it would have been nice if you tried and solve the linter error instead of disabling the linter check, it's advisable to use linter diable as a last resort